config/default/common/config/metadata/layers/multi-mission/hls/HLS_Color_Infrared_Landsat.md

```diff
@@ -2,7 +2,7 @@
 
 The dynamically generated Reflectance (Bands 5-4-3, False Color) imagery layer is a false color composite that is used to gauge plant health and assess plant density. Plants reflect in near infrared and green light, while absorbing in red. Areas with better vegetation health appear red, and denser plant growth is a darker red. Cities, urban areas, and exposed ground appear grey or tan and water is blue or black.
 
-The Reflectance imagery layer from Landsat 8 and 9/OLI product (L30) is available through the HLS project from the Operational Land Imager (OLI) aboard the Landsat 8 and 9 satellites. The sensor resolution is 30 m, imagery resolution is 30 m, and the temporal resolution is daily with a 16 day revisit time. The imagery is available in Worldview/GIBS approximately 2 - 4 days after satellite overpass. There is a separate combined Sentinel 2A and Sentinel 2B imagery layer available.
+The Reflectance imagery layer from Landsat 8 and 9/OLI product (L30) is available through the HLS project from the Operational Land Imager (OLI) aboard the Landsat 8 and 9 satellites. The sensor resolution is 30 m, imagery resolution is 30 m, and the temporal resolution is daily with a 8 day revisit time. The imagery is available in Worldview/GIBS approximately 2 - 4 days after satellite overpass. There is a separate combined Sentinel 2A and Sentinel 2B imagery layer available.
```

Comment:
In each instance of "...resolution is daily with a 8 day revisit time" "a" should be "an". (Sorry)

Reply:
Fixed!